standup/status/migrations/0017_fix_replies.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-05-25 00:09
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def fix_replies(apps, schema_editor):
+    """Adds 'In reply to url: ' to beginning of all replies"""
+    Status = apps.get_model('status', 'Status')
+
+    replies = Status.objects.filter(reply_to__isnull=False)
+    for reply in replies:
+        content = reply.content
+        content = 'In reply to https://www.standu.ps/status/%d/:\n%s' % (
```

Comment:
I'm not sure this is the best thing ever, but it's good enough. There aren't many replies, so even if this isn't the best thing we could do, it feels like a "meh--let's just move on" issue.